modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -192,27 +195,28 @@ export async function sendShareToBitgo(
 
   switch (shareType) {
     case SendShareType.KShare:
+      assert(signerShare, `signer share must be present`);
```

Comment:
Also I can see that signerShare also sent out in MuShare block also, so would the assert applicable for both the cases?

Reply:
signerShare is only required when sending K share i.e. the first part of the signing. at other stage signer share could be whatever.